internal/storage/debug_bucket.go

```diff
@@ -159,12 +159,67 @@ func (b *debugBucket) CreateObject(
 	return
 }
 
+///////////////////////////////////////////////////////////////////////////////////////
+//////////// Chunk Uploader ///////////////////////////////////////////////////////////
+///////////////////////////////////////////////////////////////////////////////////////
+
+type debugChunkUploader struct {
+	gcs.ChunkUploader
+	bucket    *debugBucket
+	requestID uint64
+	desc      string
+	startTime time.Time
+	wrapped   gcs.ChunkUploader
+}
+
+func (dcu *debugChunkUploader) Upload(ctx context.Context, contents io.Reader) error {
+	err := dcu.wrapped.Upload(ctx, contents)
+	// Don't log EOF errors, which are par for the course.
+	if err != nil && err != io.EOF {
```

Comment:
we will not get io.IOF errors from upload, right ? if yes, can we remove this check ?

Reply:
Yes, removed.